.github/workflows/build.yaml

```diff
@@ -81,14 +81,56 @@ jobs:
           username: ${{ secrets.QUAY_STACKROX_IO_RW_USERNAME }}
           password: ${{ secrets.QUAY_STACKROX_IO_RW_PASSWORD }}
 
-      - name: Build and push OCI image
+      - name: Build and push OCI amd64 image
         if: github.event_name != 'pull_request'
         uses: docker/build-push-action@v5
         with:
           context: .
           push: ${{ github.event_name != 'pull_request' }}
-          tags: ${{ steps.meta.outputs.tags }}
+          tags: ${{ steps.meta.outputs.tags }}-amd64
           labels: ${{ steps.meta.outputs.labels }}
+          platforms: linux/amd64
+          build-args: |
+            ARCH=amd64
+
+      - name: Build and push OCI arm64 image
+        if: github.event_name != 'pull_request'
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.meta.outputs.tags }}-arm64
+          labels: ${{ steps.meta.outputs.labels }}
+          platforms: linux/arm64
+          build-args: |
+            ARCH=arm64
+
+      - name: Build and push OCI ppc64le image
+        if: github.event_name != 'pull_request'
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.meta.outputs.tags }}-ppc64le
+          labels: ${{ steps.meta.outputs.labels }}
+          platforms: linux/ppc64le
+          build-args: |
+            ARCH=ppc64le
+
+      - name: Build and push OCI 390x image
+        if: github.event_name != 'pull_request'
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          push: ${{ github.event_name != 'pull_request' }}
+          tags: ${{ steps.meta.outputs.tags }}-s390x
+          labels: ${{ steps.meta.outputs.labels }}
+          platforms: linux/s390x
+          build-args: |
+            ARCH=s390x
```

Comment:
Would it be easy to run these 4 tasks in parallel? Not a big deal if not, this should run pretty quick.

Reply:
Since it they are "steps" im not sure if they can run in parallel, would need to refactor it to be seperate jobs i believe. If needed i can look into it